src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -114,6 +114,11 @@ const ProjectsTable: React.FC = () => {
     }
   ];
 
+  const filterValues = JSON.parse(
+    // sets filter to a default value if no filter is stored in local storage
+    localStorage.getItem('projectsTableFilters') ?? '{"columnField": "carNumber", "operatorValue": "=", "value": ""}'
```

Comment:
I would change 'projectsTableFilters' to 'projectTableFilter', in order to make it clear that there can only be one projects table filter applied at a time.